### ISSUE TITLE
`ConstantCartesianCoriolis` test dominates GPU testing wall time

### ISSUE THREAD

**Issue by simone-silvestri** [CliMA/Oceananigans.jl#2647]
the `inertial_oscillations_work_with_rotation_in_different_axis` (in `test_dynamics.jl`) takes about 1.5 hours to complete on a GPU, despite testing only slanted coriolis force.

It would be probably better to skip it or design a slimmer version of the test...

**Comment by tomchor** [comment]
Good catch! I think we should test that, but indeed a slimmer version of that test is preferred.

**Comment by navidcy** [comment]
Why though it takes 1.5hrs? Any ideas?

**Comment by tomchor** [comment]
> Why though it takes 1.5hrs? Any ideas?

Not sure. It runs two models for over 3000 time steps, but the GPU should make that pretty fast. The only "unusual" thing I see with that example is that it's a 0d model, and maybe that doesn't play well with the GPU?

In any case, maybe it's easier to just test that a model with `ConstantCartesianCoriolis()` returns the correct tendencies instead of running two models for a whole inertial period?
